pkg/bloomgateway/client_test.go

```diff
@@ -24,7 +24,7 @@ func TestBloomGatewayClient(t *testing.T) {
 	flagext.DefaultValues(&cfg)
 
 	t.Run("", func(t *testing.T) {
-		_, err := NewGatewayClient(cfg, l, reg, logger)
+		_, err := NewGatewayClient(cfg, l, reg, logger, "metricsNamespace")
```

Comment:
nit: `s/metricsNamespace/loki/` please ignore if this is intentional

Reply:
Yes, this is a good idea. It's changed now.